File: Iniciante/URI 1015.cs
using System; 
 
class URI {
     
    static void Main(string[] args) { 
        string[] input;
        
        input = Console.ReadLine().Split(' ');
        double x1 = double.Parse(input[0]);
        double y1 = double.Parse(input[1]);
        
        input = Console.ReadLine().Split(' ');
        double x2 = double.Parse(input[0]);
        double y2  = double.Parse(input[1]);
        
        double distancia = Math.Sqrt(Math.Pow((x2 - x1), 2.0) + Math.Pow((y2 - y1), 2.0));
        Console.WriteLine("{0:F4}", distancia);
    }
    
}

File: Iniciante/URI 1015.py
import math

distancia = None
y1 = None
y2 = None
x1 = None
x2 = None
lista2 = None
lista1 = None

def read_line():
  try:
    # read for Python 2.x
    return raw_input()
  except NameError:
    # read for Python 3.x
    return input()

lista1 = read_line().split(" ")
lista2 = read_line().split(" ")
x1 = float((lista1[0]))
y1 = float((lista1[1]))
x2 = float((lista2[0]))
y2 = float((lista2[1]))
distancia = math.sqrt(((math.pow((x2 - x1), 2)) + (math.pow((y2 - y1), 2))))
print(str("{:0.4f}".format(distancia)) + str(''))
